### DISCUSSION TITLE
Have you tried Revi, GhostSpectre or Windows X Lite? What do you love about Atlas?

### DISCUSSION THREAD

**Discussion by grand-lotus-iroh** [meetrevision/playbook#122] [General]
Just burned up a couple hours checking out all the custom forks of Windows. I really like the Windows 10 builds across all versions, everything about Windows 11 feels slower, but it sure does look pretty.

I don't mind the custom iso installs from GS / WXL but I feel like Atlas across the board for W10 is the most snappy and least lag. It does have some printer spooling issues that take some time to fix.

**Comment by melo936** [comment]
???

**Reply by TecnikOfficial** [reply]
REVIOS gives performance with stability whereas atlas os strips many things (including security) to squeeze out performance i cant use it as daily driver as its unstable and app combability is not great . if only purpose is to play games then yes atlas os do hav advantage @grand-lotus-iroh
